xbmc/guilib/GUITextLayout.h

```diff
@@ -64,7 +64,7 @@ class CGUIString
 class CGUITextLayout
 {
 public:
-  CGUITextLayout(CGUIFont *font, bool wrap, float fHeight=0.0f, CGUIFont *borderFont = NULL);  // this may need changing - we may just use this class to replace CLabelInfo completely
+  CGUITextLayout(CGUIFont *font, bool wrap, float fHeight=0.0f, CGUIFont *borderFont = NULL, UTILS::Color bgcolor = UTILS::COLOR::TRANSPARENT);  // this may need changing - we may just use this class to replace CLabelInfo completely
```

Comment:
please avoid default parameters. those are rather hacky and when browsing code you always have to study the header or you don't understand the behaviour. it was those crappy default parameters that misguided me on the DrawQuad